src/app/Components/GlobalSearchInput/useSearch.ts

```diff
@@ -15,6 +16,12 @@ import { useThrottle } from "react-use"
 
 export const useSearch = ({ query }: { query: string }) => {
   const throttledQuery = useThrottle(query, SEARCH_THROTTLE_INTERVAL)
+  const selectedTab = useSelectedTab()
+
+  let contextScreenOwnerType = OwnerType.search
+  if (selectedTab === "home") {
+    contextScreenOwnerType = OwnerType.home
+  }
```

Comment:
I find this easier to read, but it's very subjective, and I'm also okay with the `let-if` pattern.

```suggestion
  const contextScreenOwnerType = selectedTab === "home" ? OwnerType.home : OwnerType.search
```